components/ai_chat/resources/page/components/input_box/index.tsx

```diff
@@ -249,6 +254,26 @@ function InputBox(props: InputBoxProps) {
             tabs={props.context.tabs}
             setShowAttachments={props.context.setShowAttachments}
           />
+          {props.context.hasAcceptedAgreement &&
```

Comment:
do we not have tests for this because it's part of settings?

Reply:
It's the AI Chat webui and I've added some tests